kitsune/wiki/tests/test_parser.py

```diff
@@ -297,7 +296,7 @@ def test_build_template_params_named_anonymous_numbered(self):
         eq_({'1': 'a', 'hi': 'test', '3': 'z'}, _btp(['hi=test', 'a', '3=z']))
 
     def test_unapproved_template(self):
-        document(title='Template:new').save()
+        TemplateDocumentFactory(title='Template:new')
```

Comment:
It might be worth tweaking `TemplateDocumentFactory` so that it prefixes the title with `Template:` if it's not there already.